### PR TITLE
Course page now seperates active and inactive courses

### PR DESCRIPTION
**Pull request by PeterBifone**
Resolves issue #19

### CODE REVIEW

File: src/main/routes/course.js [hunk 1]
```diff
@@ -142,10 +146,15 @@ router.route('/')
 				return (100 * completion).toFixed(1)
 			}
 		}
+		const portfolio_active = portfolio_all.filter(element => !element.read_only)
+		const portfolio_inactive = portfolio_all.filter(element => element.read_only)
+		console.log(portfolio_inactive)
```
Comment (spigo900) [inline src/main/routes/course.js]: Left debugging info in?

File: src/main/routes/course.js [hunk 2]
```diff
@@ -142,10 +146,15 @@ router.route('/')
 				return (100 * completion).toFixed(1)
 			}
 		}
+		const portfolio_active = portfolio_all.filter(element => !element.read_only)
+		const portfolio_inactive = portfolio_all.filter(element => element.read_only)
```
Comment (spigo900) [inline src/main/routes/course.js:150]: Yay, filter.

File: src/main/views/course/index.html [hunk 3]
```diff
@@ -7,16 +7,16 @@
 	<section class="active_courses">
 		<h2>Active Courses <i class="material-icons">help</i></h2>
 		<table>
-			<tr>				
+			<tr>
```
Comment (spigo900) [inline src/main/views/course/index.html:10]: As it should be.

File: src/main/views/course/index.html [hunk 4]
```diff
@@ -43,22 +42,19 @@ <h2>Archived Courses</h2>
...
-				<td><a href="/course/123">view <i class="material-icons">pageview</i></a></td>
-			</tr>
-			<tr>
-				<td>CS499</td>
-				<td>Fall</td>
-				<td>2019</td>
-				<td>12 / 12</td>
-				<td>Dec 5, 2019</td>
-				<td><a href="/course/123">view <i class="material-icons">pageview</i></a></td>
+				{{#portfolio_inactive}}
```
Comment (spigo900) [inline src/main/views/course/index.html]: Why is there no opening `<tr>` after line 45?

**Review by spigo900 — APPROVED** [review]

File: src/main/routes/course.js (hunk 1 above)
Comment (PeterBifone) [reply src/main/routes/course.js]: Whoops

**Pull request merged**